### ISSUE TITLE
The Prefix ID creates a talk page integer and not one for content. 

### ISSUE THREAD

**Issue by Alex-Allgood** [hallowelt/migrate-confluence#94]
I am having an issue with mediawiki seeing all the imported pages as talk pages. Going off of the what is on the help page in mediawiki for custom namespaces, content pages are supposed to be even integers. When I ran the tool I ended up getting a prefix ID of 6848515, is there anyway to influence the tool so that I can have it create an even number? That will allow me to properly use it as a content space and create its relevant talk space. 
https://www.mediawiki.org/wiki/Manual:Using_custom_namespaces#Content_namespaces

**Comment by Alex-Allgood** [comment]
Is there a way to force the pages to conform to a namespace ID integer that I want?

**Comment by Alex-Allgood** [comment]
Can I just find and replace the bad odd integer with what I need in the working files?

**Comment by osnard** [comment]
Yes, you can usually just use search&replace on the files created by the `analyze` step to change prefixes or apply other kinds of title changes.

**Comment by Boldewyn** [comment]
The migration tool doesn’t output numeric namespace IDs anymore. It assumes configured namespaces in the target wiki. Mapping from Confluence space to wiki namespace is done in the config:

```yaml
config:
    space-prefix:
        Confluence-space-name: Wiki-namespace
```
